src/main/java/codechicken/nei/ItemsGrid.java

```diff
@@ -92,6 +103,8 @@ public int getColumns()
 
     public void setGridSize(int mleft, int mtop, int w, int h)
     {
+        if(width != w || height != h || mleft != marginLeft || mtop != marginTop)
```

Comment:
If you open craft station with attached chest and after without it gui size don't change. this problem was then i create Items Grid

Reply:
I can't reproduce this with a Tinkers crafting station, is that what you were using?

Reply:
When you fixed the issue for invalidSlotMap you fixed and this is

Reply:
I think your message got cut off.